src/sentry/snuba/metrics/fields/base.py

```diff
@@ -328,15 +325,15 @@ def validate_can_orderby(self) -> None:
     def run_post_query_function(
         self,
         data: SnubaDataType,
-        metrics_query: MetricsQuery,
         metric_mri: str,
         alias: str,
         idx: Optional[int] = None,
+        params: Optional[MetricOperationParams] = None,
```

Comment:
wdyt if we rename it to `metric_op_params` since it's already have type `MetricOperationParams`?

Reply:
hmm I feel like no need to make it more verbose plus its convention we name it `params` in other areas of the code as well https://github.com/getsentry/sentry/blob/6307cf52c4c7f185f9023c6279e565dd7812c202/src/sentry/snuba/metrics_enhanced_performance.py#L18

Reply:
good, so resolved.